docs/developer/contributing/api-guidelines.rst

```diff
@@ -204,31 +204,38 @@ each API endpoint.
 
 A generic ``sort`` parameter **must** be used to describe sorting rules. If
 there are multiple fields that a collection can be sorted by, the ``sort``
-parameter **must** accept a list of comma separated fields, indicating
-descending sort priority. A minus sign implies descending sort order in the
-subsequent field.
+parameter **must** accept a list of comma separated fields. The format of the
+``sort`` parameter values has to follow: ``field:ASC|DESC``.
 
 - Retrieving all files, ordered alphabetically by file type and by descending
   file size:
-  ``GET /files?sort=type,-size``
+  ``GET /files?sort=type:ASC,size:DESC``
 
 6.3 Pagination
 ~~~~~~~~~~~~~~
 
 Pagination **must** be supported by all endpoints returning more than just
-a single resource. Pagination **must** implemented using the query
-parameters ``page`` and ``per_page``. Link headers **must** be included in the
-response indicating:
+a single resource. Pagination **must** be implemented using the query
+parameters ``page`` and ``per_page``. Both the parameters have to accept
+any positive integer value and default to 1 and 20 accordingly. Non-positive
```

Comment:
Should we leave the default page size open so that it can be set specifically for each case?

Reply:
Yes, why not. Let me tweak that.